docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -314,11 +314,11 @@ receiving warnings that the ETH1 node is unavailable.
     eth1-endpoint: "http://localhost:8545"
     ```
 
-The JSON-RPC URL of Ethereum 1.0 node.
+The JSON-RPC URL of Ethereum 1.0 node. This option must be specified if running a validator.
 
-If not specified, then provide an initial state using the [`--initial-state`](#initial-state) option, or
-start Teku from an existing database using [`--data-path`](#data-path), which provides the initial
-state to work from.
+If not specified (that is, you're running a beacon chain client only), then provide an initial state
+using the [`--initial-state`](#initial-state) option, or start Teku from an existing database using
+[`--data-path`](#data-path), which provides the initial state to work from.
```

Comment:
If you're running the already started public networks like MainNet or Pyrmont then you don't need to provide an initial state either - Teku already has it built in.

Reply:
Implemented suggestion.

Reply:
Thanks @ajsutton, could you please approve this if you agree with the change?

Reply:
Sorry, beat me to it :)